app/views/petitions/check_results.html.erb

```diff
@@ -1,23 +1,22 @@
 <% content_for(:page_title) { "Create a new e-petition - e-petitions" } %>
-<% content_for :title do %>
-<div class="title_block">
-  <h1>Create a new e-petition</h1>
-</div>
-<% end %>
 
-<%= render "pre_creation_search_form" %>
+<%= link_to check_petitions_path, class: "back-page" do %>
+  &#9664; Back <span class="visuallyhidden"> to previous page</span>
```

Comment:
What's the point of the visually hidden bit?  I guess it's for more context about what the "back" means for screen readers, but it's not like it provides much noise for others - could just leave it in?  Which would neatly sidestep the fact that the "back" buttons on the forms are ``<input>`` elements, not ``<button>`` elements and so can't have markup in their values, so we can't do the same "hidden text for context" trick with them.